jupyter_rfb/widget.py

```diff
@@ -40,6 +39,10 @@ class RemoteFrameBuffer(ipywidgets.DOMWidget):
     * *css_width*: the logical width of the frame as a CSS string. Default '500px'.
     * *css_height*: the logical height of the frame as a CSS string. Default '300px'.
     * *resizable*: whether the frame can be manually resized. Default True.
+    * *quality*: the quality of the JPEG encoding during interaction/animation
+      as a number between 1 and 100. Default 80. Set to lower numbers for more
+      performance on slow connections. Set to 100 for lossless (PNG),
+      but note that each interaction is ended with a lossless image anyway.
```

Comment:
Is this clear?

Is the name "quality" clear enough or can we make it more specific?

Reply:
Looks like Pillow and GDAL (in the JPEG driver and GeoTIFF driver) refer to this as "quality" or "JPEG_QUALITY" so it might be good enough to just leave it. Or call it jpeg_quality?

Is pillow a hard requirement? Simplejpeg is currently optional, right? What happens if neither of these is installed? Should that be mentioned here? Should this be rephrased with "If set to 100 or if JPEG encoding isn't possible (missing pillow or simplejpeg dependencies), then lossless PNGs will always be sent."

Reply:
Neither is a hard requirement, and if both are missing then PNG is used. Good point - I forgot to include that info here.